test/unit/org/apache/cassandra/index/sai/cql/types/IndexingTypeSupport.java

```diff
@@ -74,7 +74,7 @@ public void createTable()
     {
         dataset.init();
 
-        createTable(String.format("CREATE TABLE %%s (pk int, ck int, value %s, PRIMARY KEY(pk, ck))", dataset));
+        createTable(String.format("CREATE TABLE %%s (pk int, ck int, value %s, PRIMARY KEY(pk, ck)) WITH CLUSTERING ORDER BY (ck desc)", dataset));
```

Comment:
I'm not sure if this is the right place to test the `indexTermType.isReversed()` logic.

Reply:
And, I've yet to modify `RandomIntersectionTester`.

Reply:
Resolved both of these on Slack